doc/ref_model/chapters/chapter03.md

```diff
@@ -258,6 +260,22 @@ Hardware Accelerators can be used to offload software execution for purpose of a
 
 More details about Hardware Acceleration are in [Section 3.8 Hardware Acceleration Abstraction](chapter03.md#3.8).
 
+<a name="3.4.2"></a>
+### 3.4.2 Time Sensitive Networking
+
+Many network function have time sensativity for processing and require high precision synchronized clock for the virtialization/containerization platform.  Subset of these, like RAN, in addition require support for Synchronous Ethernet as well.
+
+| Reason for using Synchronous Precision Clock | Example | Comment |
+|---|---|---|
+| Achieve technical requirements | Strict latency or timing accuracy | Must be done for precise low latency communication between data source and receiver |
+| Achieve technical requirements | Separation of processing pipeline | Ability to seperate RAN into RU, DU, CU on different or stretch clusters |
+| Improve cost/felxibility | Ability to innovate on different portions of Radio Network Functoins | Different latency requirements for different parts of RAN processing |
```

Comment:
Thinking if this row is really needed, if using Synchronous Precision Clock as such improves cost/flexibility? Those that require it are already in "Achieve technical requirements". Those that don't need it would not be mentioned as Precision cannot optionally help them.
If needed: "felxibility" -> "flexibility", "Functoins" -> "Functions".

Reply:
done